app/interactor/users.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('../logger');
+const { generateToken, comparePass } = require('../helpers/users');
+const { findUserByEmail } = require('../services/users');
+const { tokenError, authenticationError } = require('../errors');
+const { GENERATE_TOKEN, AUTHENTICATION } = require('../../config/constants/errorMessages');
+
+exports.createToken = async (req, next) => {
```

Comment:
This looks like a middleware. It is better to handle interactors as inmutable functions, avoiding the access to `req` and `next` and just returning the expected response or throwing an error. The next call at errors will remain in the controller.
It is just for convention 😛